src/date/__tests__/monthsAfterStr-test.js

```diff
@@ -0,0 +1,10 @@
+import { expect } from 'chai';
+import monthsAfterStr from '../monthsAfterStr';
+
+describe('monthsAfterStr', () => {
+    it('should return a date string with given number of months ahead of the current date', () => {
```

Comment:
Have actually added to accommodate the issue of default starting date in one of our date type control which i noticed we used a date strings, excludefrom of selfExclusion to be precise. As for the naming , i tried to make the word after a similar file "oneYearAfterStr" .  The same goes for the test wording , we can change that to what seems more clear though anytime . 
`new Date(new Date().setMonth(new Date().getMonth() + m))` . It can easily we proven , just copy to your browser console and test :wink: .